### PR TITLE
Tweaked styles

### PR DESCRIPTION
**Pull request by pppemo**

### CODE REVIEW

**Comment by ugogo** [comment]
@pppemo Could you provide more context? Why do you need this?

**Comment by pppemo** [comment]
Because if this component is placed in another component that has:
```
text-align: center

f-Button: {
width: 100%
}
```

this is what happens
image

File: packages/css/src/components/Alert.css [hunk 1]
```diff
@@ -11,6 +11,7 @@
 }
 
 .f-Alert {
+  text-align: left;
```
Comment (ugogo) [inline packages/css/src/components/Alert.css]: Move this line after the variables

Comment (pppemo) [reply packages/css/src/components/Alert.css]: done

**Pull request merged**